debug/main.py

```diff
@@ -21,8 +25,9 @@ def cli():
     """SONiC command line - 'debug' command"""
     pass
 
-
-p = subprocess.check_output(["sudo vtysh -c 'show version'"], shell=True, text=True)
+debug_bgp = "debug bgp "
```

Comment:
>debug_bgp [](http://example.com/codeflow?start=0&length=9)

Most of "debug bgp " is used in static string, if you extract, you need use `+` to concat. Is it more readable to keep original long string?